docs/sphinx/user_guide/tutorial/sort.rst

```diff
@@ -0,0 +1,182 @@
+.. ##
+.. ## Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+.. ## and RAJA project contributors. See the RAJA/COPYRIGHT file
+.. ## for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+.. ##
+
+.. _sort-label:
+
+--------------------------------------------------
```

Comment:
@MrBurmark this section looks good overall.  The only suggestion I have would be to replace one or two of the examples with something that uses different comparison operators. All examples here use 'less'.

Reply:
the last section should have greater than examples and sort_pairs examples, but I'll add a greater example earlier.